datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/datajob/mappers/DataJobMapper.java

```diff
@@ -75,6 +76,9 @@ public DataJob apply(@Nonnull final EntityResponse entityResponse) {
             } else if (DATA_JOB_INPUT_OUTPUT_ASPECT_NAME.equals(name)) {
                 final com.linkedin.datajob.DataJobInputOutput gmsDataJobInputOutput = new com.linkedin.datajob.DataJobInputOutput(data);
                 result.setInputOutput(mapDataJobInputOutput(gmsDataJobInputOutput));
+                if (gmsDataJobInputOutput.hasFineGrainedLineages() && gmsDataJobInputOutput.getFineGrainedLineages() != null) {
```

Comment:
nitpick: the first is sufficient, but the second intellij will make you do

Reply:
ahh so if `hasFineGrainedLineages` is true, then we know it won't be `null`? that's what i thought but intellij did indeed get annoyed with me